src/main/js/main_html5.js

```diff
@@ -150,7 +150,7 @@ require("../../../html5-common/js/utils/environment.js");
     var firstPlay = true;
     var playerDimension = dimension;
     var videoDimension = {height: 0, width: 0};
-    var queuedInitialTime = 0;
+    var initialTime = { value: 0, reached: true };
```

Comment:
There was a bug where the seeked events were being raised when we were setting initialTime (PBW-4407)